### PR TITLE
Jiden/langium interactive example

### PR DESCRIPTION
**Pull request by jonah-iden**
an interactive example for sprotty for the Statemachine language. 
This is using the monaco editor with a Langium language server which is implementing a sprotyy diagram server through the sprotty-langium integration. 
communication is realized through a custom modelSource using the monacoLanguageClient 

**How to test**
1. npm install
2. npm run build --workspaces
3. npm start
4. example should be available under [http://localhost:8080/showcase](http://localhost:8080/showcase)

### CODE REVIEW

File: showcase/language-configuration.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (spoenemann) [inline showcase/language-configuration.json:1]: We're not using the language in vscode, so you can remove this file.

File: showcase/syntaxes/statemachine.tmLanguage.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (spoenemann) [inline showcase/syntaxes/statemachine.tmLanguage.json:1]: We're not using the language in vscode, so you can remove this file.

**Review by spoenemann — COMMENTED** [review]
Thanks!

Please remove the generated files in `static-dev` from version control and add them to `.gitignore`

File: .gitpod.yml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 tasks:
-  - init: npm install
+  - before: npm install
+    init: npm run build --workspaces
```
Comment (spoenemann) [inline .gitpod.yml:3]: `before` is meant to install something globally. In this case I'd write `init: npm install && npm run build --workspaces`

**Review by spoenemann — APPROVED** [review]

File: .gitpod.yml (hunk 3 above)
Comment (jonah-iden) [reply .gitpod.yml:3]: oh ok sorry, i didn't knew that

**Pull request merged**